src/SDKs/TrafficManager/TrafficManager.Tests/project.json

```diff
@@ -0,0 +1,40 @@
+{
+  "version": "1.0.0-*",
```

Comment:
I believe we do not allow project.json and xproj files.

Reply:
Removed both.